docs/reports/mapped_deprecated.md

```diff
@@ -1,13 +1,13 @@
 # Mapped deprecated terms
 | Ontology                                    |   Tot deprecated in Mondo |
 |:--------------------------------------------|--------------------------:|
-| [OMIM](./mapped_deprecated_omim.md)         |                        40 |
-| [DOID](./mapped_deprecated_doid.md)         |                         4 |
-| [NCIT](./mapped_deprecated_ncit.md)         |                         3 |
-| [ORDO](./mapped_deprecated_ordo.md)         |                       113 |
-| [ICD10WHO](./mapped_deprecated_icd10who.md) |                         0 |
-| [ICD10CM](./mapped_deprecated_icd10cm.md)   |                         0 |
+| [ORDO](./mapped_deprecated_ordo.md)         |                       574 |
```

Comment:
@matentzn I expected all numbers to decrease after this change, not increase. But I face checked and the ones I looked at were indeed obsolete in Orphanet but not yet so in Mondo.

Reply:
@matentzn I sent you a message [on slack](https://monarchinitiative.slack.com/archives/D02CT4EGR8B/p1687309026929559). You are correct. All of your comments here are reflective of the fact that I have old inputs. Let me know if you'd like me to run the full pipeline (c). If not, I can do (a), just the code changes, and reset these outputs artefects.

Reply:
@matentzn Looks like since you ran the pipeline and did the merge, this PR is now set up according to what you preferred on slack (option 'b'). But I should re-run it to get the correct outputs. Will do that after vacay.

Reply:
Thanks for merging!